src/Umbraco.Web.UI.Client/src/views/propertyeditors/listview/orderDirection.prevalues.html

```diff
@@ -1,15 +1,8 @@
 ﻿<div>
     <ng-form name="listViewOrderDirectionForm">
-        <label class="radio">
-            <input type="radio" name="orderDirection" value="asc" ng-model="model.value" />
-            Ascending <code>[a-z]</code>
-        </label>
-        <label class="radio">
-            <input type="radio" name="orderDirection" value="desc" ng-model="model.value" />
-            Descending <code>[z-a]</code>
-        </label>
-
-        <span ng-messages="listViewOrderDirectionForm.orderDirection.$error" show-validation-on-submit >
+        <umb-radiobutton name="orderDirection" value="asc" model="model.value" text="Ascending"></umb-radiobutton>
```

Comment:
This removes `<code>[a-z]</code>`, which should be preserved. Same for the `Descending` radiobutton below it.

Reply:
@poornimanayar change this part to something like this:

```
<div class="flex items-center">
    <div class="flex-inline items-center">
        <umb-radiobutton name="orderDirection" value="asc" model="model.value" text="Ascending" /><code class="ml1">[a-z]</code>
    </div>
    <div class="flex-inline items-center ml3">
        <umb-radiobutton name="orderDirection" value="desc" model="model.value" text="Descending" /><code class="ml1">[z-a]</code>
    </div>
</div>
```